### DISCUSSION TITLE
Can FEPX be compiled with mpich?

### DISCUSSION THREAD

**Discussion by helsmy** [neperfepx/FEPX#97] [Q&A] [answered]
Can FEPX be compiled with mpich?  I have compiled and installed mpich into my system. It is diffcult to clear all mpich files and switch to openmpi. And when I compiled with mpich, I got error: "Can't open included file 'mpif-config.h'".

**Comment by mkasemer** [comment, marked as answer]
Currently we only test FEPX with OpenMPI, as described in the documentation. If you are interested in modifying FEPX to compile using MPICH, it would be necessary to edit the code (particularly `libparallel`) to allow this.

**Reply by helsmy** [reply]
Thanks. I am not very familiar to  fortran. I'll try to install openmpi in a clean enviroment.
